### PR TITLE
#25 functionality to send secure fields data without storing in the notifier database

### PR DESCRIPTION
**Pull request by Syuziko**

### CODE REVIEW

**Comment by dnavre** [comment]
Had a look at it, everything looked okay. Would be great if @mikron  also manager to have a look.

File: infra/infra-integrations/infra-integrations-email/src/main/java/com/sflpro/notifier/externalclients/email/mandrill/communicator/MandrillApiCommunicatorImpl.java [hunk 1]
```diff
@@ -47,7 +47,7 @@ public MandrillApiCommunicatorImpl() {
         LOGGER.debug("Initializing Mandrill web service communicator");
     }
 
-    private void chkMandrillApi() {
+    private void checkMandrillApi() {
```
Comment (mikron) [inline infra/infra-integrations/infra-integrations-email/src/main/java/com/sflpro/notifier/externalclients/email/mandrill/communicator/MandrillApiCommunicatorImpl.java]: chk is better than check

Comment (dnavre) [reply infra/infra-integrations/infra-integrations-email/src/main/java/com/sflpro/notifier/externalclients/email/mandrill/communicator/MandrillApiCommunicatorImpl.java]: oh, c'mon :D

File: core/core-services/src/main/java/com/sflpro/notifier/services/notification/dto/NotificationDto.java [hunk 2]
```diff
@@ -67,13 +69,22 @@ public void setClientIpAddress(final String clientIpAddress) {
         this.clientIpAddress = clientIpAddress;
     }
 
+    public boolean isHasSecureProperties() {
```
Comment (dnavre) [inline core/core-services/src/main/java/com/sflpro/notifier/services/notification/dto/NotificationDto.java]: Remove the `is` prefix. Sorry for being a grammar-nazi.

**Pull request merged**